src/test/starknet.t.sol

```diff
@@ -37,7 +37,7 @@ contract ConfigStarknet {
 
         starknetValues = StarknetValues({
             l2_spell:                  0,  // Set to zero if no spell is set.
-            core_implementation:       0x3CeCEe6B359fAc09c79Ca4032826894F7e660B33, // As at 2023-05-17
+            core_implementation:       0x55bBBE4C8E426Be2Ec6F5b05A1728f9AFCa1Eb5C, // As at 2024-03-13
```

Comment:
Note: independently from this PR, implementation of the `STARKNET_CORE` has changed recently, therefore, I had to update this address for the starknet tests to pass. The test suggests to "inspect the new implementation for safety": https://github.com/makerdao/spells-goerli/blob/afc5a201372e8de0a954c7dccff743788307963d/src/test/starknet.t.sol#L191
Here is the diff between the new and the old contract: https://www.diffchecker.com/gxLc9kdN/